src/components/sdfgViewer.ts

```diff
@@ -237,12 +273,33 @@ export class SdfgViewerProvider
                 });
                 break;
             case 'go_to_sdfg':
+                const msgs = [];
+                if (message.zoom_to) {
+                    msgs.push(
+                        new Message(message.sdfg_name, {
+                            type: 'zoom_to_node',
+                            uuid: message.zoom_to,
+                        })
+                    );
+                }
+                if (message.display_bps) {
+                    msgs.push(
+                        new Message(message.sdfg_name, {
+                            type: 'display_breakpoints',
+                            display: message.display_bps,
+                        })
+                    );
+                }
                 SdfgViewerProvider.getInstance()?.openViewer(
                     vscode.Uri.file(message.path),
-                    message.zoom_to,
-                    message.display_bps
+                    msgs
                 );
                 break;
+            case 'handle_messages':
```

Comment:
Thoughts on renaming this to something like 'process_queued_messages'?

Reply:
yeah that is better